configs/btc/registration_testnet.properties

```diff
@@ -1,16 +1,17 @@
 # ========= Bitcoin Registration Service =========
 btc-registration.registrationAccount=btc_registration_service_red@notary
 btc-registration.mstRegistrationAccount=mst_btc_registration_service_red@notary
+btc-registration.notaryAccount=notary_red@notary
 # port of registration service
 btc-registration.port=8084
-# --------- Credetnials ------
-btc-registration.registrationCredential.accountId=notary_red@notary
+# ---------- Credentials ---------
+btc-registration.registrationCredential.accountId=btc_registration_service_red@notary
```

Comment:
please, add keypair for btc_registration_service_red

Reply:
this will be implemented in separate task